components/Alert/alert.css

```diff
@@ -0,0 +1,36 @@
+.alert-container {
+  box-shadow: 0 0 1px rgb(0 0 0 / 10%), 0 2px 6px rgb(0 0 0 / 10%);
+  padding: 10px;
+  max-width: 30rem;
+}
+
+.alert {
+  padding: 1rem 1rem;
```

Comment:
You could use shorthand property here as both top and bottom margin are same. `padding: 1rem`